authbridge/demos/github-issue/demo-aiac.md

```
@@ -0,0 +1,862 @@
+
+**Verify injected containers:**
+
+Confirm that the webhook injected the combined AuthBridge sidecar:
+
+```bash
+kubectl get pod -n team1 -l app.kubernetes.io/name=git-issue-agent \
+  -o jsonpath='{.items[0].spec.containers[*].name}'
+```
+
+Expected (proxy-sidecar mode, the cluster default):
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)